Dockerfile

```diff
@@ -74,11 +75,12 @@ WORKDIR /srv/sylius
 
 # build for production
 ARG APP_ENV=prod
+ARG APP_SECRET=thisvariableissuddenlyneededhere
 
 # prevent the reinstallation of vendors at every changes in the source code
 COPY composer.json composer.lock symfony.lock ./
 RUN set -eux; \
-	composer install --prefer-dist --no-autoloader --no-scripts --no-progress --no-suggest; \
+    composer install --prefer-dist --no-autoloader --no-scripts --no-progress --no-suggest; \
```

Comment:
@pamil You should install the EditorConfig plugin for your IDE to avoid whitespace problems like this in the future.